web/templates/layout/app.html.slim

```diff
@@ -0,0 +1,31 @@
+doctype html
+html lang="en"
+  head
+    meta charset="utf-8"
+    meta content="IE=edge" http-equiv="X-UA-Compatible"
+    meta content="width=device-width, initial-scale=1" name="viewport"
+    meta content="" name="description"
+    meta content="" name="author"
+    title Hello ApplicationName!
+    link rel="stylesheet" href="#{static_path(@conn, "/css/app.css")}"
+
+  body
+    .container
+      header.header
+        span.logo
+        nav role="navigation"
+          ul.nav.nav-pills.pull-right
+          = if logged_in?(@conn) do
```

Comment:
Alam ko may kung anong magic si Elixir na pag nasa templates ka may `@current_user` na object. Macro yun na kukuhain niya yung nasa session mo `@current_user`. Though not sure kung nasa session ba o nasa assigns ni plug yung kukuhain niya. Might as well try. Kung meron, this can be rewritten as:
```
if @current_user do 
... 
end
```